AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h

```diff
@@ -38,7 +38,7 @@ NS_SWIFT_NAME(ReleaseDetails)
 /**
  * The flag that indicates whether the release is a mandatory update or not.
  */
-@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate NS_SWIFT_NAME(mandatoryUpdate);
+@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate;
```

Comment:
Same

Reply:
It's not readonly, so it's `mandatoryUpdate` in Swift.

Reply:
`NS_SWIFT_NAME` is required then